docs/src/pages/demos/tables/CustomizedTable.js

```diff
@@ -58,9 +58,9 @@ function CustomizedTable(props) {
           <TableRow>
             <CustomTableCell>Dessert (100g serving)</CustomTableCell>
             <CustomTableCell align="right">Calories</CustomTableCell>
-            <CustomTableCell align="right">Fat (g)</CustomTableCell>
-            <CustomTableCell align="right">Carbs (g)</CustomTableCell>
-            <CustomTableCell align="right">Protein (g)</CustomTableCell>
+            <CustomTableCell align="right">Fat&nbsp;(g)</CustomTableCell>
```

Comment:
So, there is 4 ways to do the same thing:
1. https://github.com/mui-org/material-ui/blob/bbfbb0ca6917475e196efc21e826e868d630e75e/packages/material-ui/src/FormLabel/FormLabel.js#L88
2. https://github.com/mui-org/material-ui/blob/bbfbb0ca6917475e196efc21e826e868d630e75e/docs/src/modules/utils/generateMarkdown.js#L258
3. `&nbsp;`
4. ` `

Reply:
I think the unicode symbol in `generateMarkdown` creates the same issue. With a basic text editor you won't see what's happening. Using `&nbsp;` should be applied there too.

I'm ok with the `FormLabel` since it's apparent what is applied but we can go with the html entity for consistency.

Reply:
I'm happy to use `&nbsp;` everywhere if possible.

Reply:
>   (Alt + Space)

I guess this is macOS-only and generates `\u00a0`